src/main/resources/ath-container/Dockerfile

```diff
@@ -33,14 +33,16 @@ RUN apt-get update && \
 
 # Install a fixed firefox version that is known to work with the current selenium version, copied from https://hub.docker.com/r/selenium/node-firefox/dockerfile
 ARG FIREFOX_VERSION=108.0.2
-RUN apt-get -y install libgtk-3-0 libasound2 libdbus-glib-1-2 libx11-xcb-dev libpci-dev libgl1-mesa-glx wget bzip2 \
+# Package versions remain unpinned
```

Comment:
```suggestion
# hadolint: Do not pin packages as they are requirements only for firefox installation (and not ofr the rest of the ATH) and the effort required to track each version would be too huge
```
WDYT?

- Prefix by hadolint for reader ot understand it's related to the below comment
- Explain the "why" instead of the "what" => are you ok with this "why" or is it another reason?

Reply:
Sounds good to me :)